src/lib/api/mtg-cardfinder.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+const https = require('https');
+const Stream = require('stream').Transform;
+const fs = require('fs');
+
+const CARDFINDER_REQUEST_URL = 'https://forums.goodgamery.com/includes/mtg/mtg_helper_cardfinder_v3.php?img=true&find=';
```

Comment:
In the ggbot repo I have a file that does this same thing but hitting gather directly. It's probably more maintainable: https://github.com/mdiehr/DiscordBot/blob/master/plugins/MTG/mtg.js